osbs/api.py

```diff
@@ -200,10 +200,18 @@ def get_build_logs(self, build_id, follow=False, namespace=DEFAULT_NAMESPACE):
             if ex.status_code != 404:
                 raise
         else:
-            if build["status"] in ["Complete", "Failed"]:
-                return build["metadata"]["labels"]["logs"]
-            else:
-                return self.os.logs(build_id, follow=False, namespace=namespace)
+            build_response = BuildResponse(build)
+            # Should this be 'build_response.is_finished()'?
```

Comment:
how about `BUILD_FINISHED_STATES`; I feel like that's subset (not really sure what will happen when one will try to get logs for cancelled/errored build)